OWNERS_ALIASES

```diff
@@ -0,0 +1,12 @@
+# See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#OWNERS_ALIASES
+
+aliases:
```

Comment:
> please also put @staebler and @rajatchopra here

This PR is already in the merge queue, so instead of pulling it out, I'll just file a follow-up PR adding them.

Reply:
sgtm